src/v2/Components/v2/ArtworkFilter/ArtworkFilters/SizeFilter.tsx

```diff
@@ -26,10 +33,15 @@ export const SizeFilter: React.FC = () => {
     filterContext.setFilter("sizes", sizes)
   }
 
+  const tokens = useThemeConfig({
+    v2: { my: 0.5, secondaryVariant: "small" as TextVariant },
+    v3: { my: 1, secondaryVariant: "xs" as TextVariant },
+  })
```

Comment:
So these are all temporary shims, and eventually for example L44 becomes just `variant="xs"` and so on, yeah?

Reply:
Exactly right. Any changes are encapsulated either in this hook or if needing to switch inside of a styled-component `getThemeConfig` and then once we transition we just inline the v3 values and that's that.